constants/styles.js

```diff
@@ -0,0 +1,28 @@
+export const BASE_STYLES = {
+  NORMAL: "NORMAL",
+  BOLD: "BOLD",
+  ITALIC: "ITALIC",
+  H1: "H1",
+  H2: "H2",
+  H3: "H3",
+  H4: "H4",
+  H5: "H5",
+  CODE: "CODE",
+  LIST: "LIST",
+  HYPERLINK: "HYPERLINK",
+}
+
```

Comment:
7. Here I created two constants; BASE_STYLES stores each style we might want to express on our side, and TAG_STYLES stores styles that can be found in the html code.